### PR TITLE
ci: Adjust the crate names in the Rust release jobs

### PR DESCRIPTION
**Pull request by vadorovsky**
Remove unused crates and adjust the naming to what we have currently in the repository.

### CODE REVIEW

File: .github/workflows/release-pr.yml [hunk 1]
```diff
@@ -8,16 +8,23 @@ on:
...
-          - light-psp10in2out
-          - light-psp4in4out-app-storage
-          - light-psp2in2out-storage
-          - light-verifier-sdk
-          - macro-circom
+          - light-indexed-merkle-tree
+          - light-macros
+          - light-merkle-tree-event
+          - light-merkle-tree-reference
+          - light-protocol
```
Comment (ananas-block) [inline .github/workflows/release-pr.yml:20]: psp-protocol

File: .github/workflows/release-pr.yml [hunk 2]
```diff
@@ -8,16 +8,23 @@ on:
...
+          - light-test-utils
+          - light-user-registry
+          - light-utils
+          - light-wasm-hasher
+          - psp-account-compression
```
Comment (ananas-block) [inline .github/workflows/release-pr.yml:25]: account-compression

**Review by ananas-block — COMMENTED** [review]
apart from the renaming looks good
``psp_account_compression`` -> ``account_compression``
``light_protocol`` -> ``psp_protocol``

File: .github/workflows/release-pr.yml (hunk 1 above)
Comment (ananas-block) [reply .github/workflows/release-pr.yml:20]: @SwenSchaeferjohann made a good point that psp is not a good fit anymore, let's talk about a different prefix in team call or just use light

File: .github/workflows/light-system-programs-tests.yml [hunk 3]
```diff
@@ -39,13 +39,13 @@ jobs:
...
               "@lightprotocol/programs:test-compressed-pda"
             ]'
-          - program: light
+          - program: registry
             sub-tests: '[
-              "@lightprotocol/programs:test-light"
+              "@lightprotocol/programs:test-registry"
             ]'
-          - program: psp-compressed-token
+          - program: compressed-token
```
Comment (SwenSchaeferjohann) [inline .github/workflows/light-system-programs-tests.yml:46]: light-compressed-token ?

File: .github/workflows/release-pr.yml [hunk 4]
```diff
@@ -8,16 +8,23 @@ on:
...
+          - psp-compressed-pda
+          - psp-compressed-token
```
Comment (SwenSchaeferjohann) [inline .github/workflows/release-pr.yml:27]: seems the changes are not applied throughout yet?
psp- -> light-

plus i think we can keep `psp-account-compression` as just `account-compression`

File: scripts/push-stateless-js-idls.sh [hunk 5]
```diff
@@ -5,7 +5,12 @@ SOURCE_DIR="./target/types"
 DEST_DIR="./js"
 DEST_DIR_STATELESS="$DEST_DIR/stateless.js/src/idls"
 
-FILES_TO_COPY=("account_compression.ts" "light.ts" "psp_compressed_pda.ts" "user_registry.ts")
+FILES_TO_COPY=(
+  "light_account_compression.ts"
```
Comment (SwenSchaeferjohann) [inline scripts/push-stateless-js-idls.sh:9]: `light_account_compression` -> `account_compression` after all i think is the current consensus

File: js/stateless.js/src/idls/light_account_compression.ts [hunk 6]
```diff
@@ -1,6 +1,6 @@
-export type AccountCompression = {
+export type LightAccountCompression = {
     version: '0.3.1';
-    name: 'account_compression';
+    name: 'light_account_compression';
```
Comment (SwenSchaeferjohann) [inline js/stateless.js/src/idls/light_account_compression.ts:3]: @vadorovsky do you know why this still has the lightaccountcompression name? maybe need to regenerate idls via `build.sh` script

Comment (vadorovsky) [reply js/stateless.js/src/idls/light_account_compression.ts:3]: I had to manually remove this file.

**Review by SwenSchaeferjohann — APPROVED** [review]
thx approved

**Pull request merged**